### DISCUSSION TITLE
md-slider not emitting value in its events

### DISCUSSION THREAD

**Discussion by simonh1000** [material-components/material-web#5183] [Q&A] [answered]
I'm using Elm with material web. I've been able to use standard `onInput` event listeners on `md-outlined-text-field` and `md-select` but they are not picking up any events in `md-slider`. Moreover there is no reference to events in https://material-web.dev/components/slider/ so perhaps these work differently? If so, how do I get the value after a change?

**Comment by datvm** [comment, marked as answer]
They works pretty well for me for both `input` and `change` events: [Lit Playground](https://lit.dev/playground/#project=W3sibmFtZSI6InNpbXBsZS1ncmVldGluZy50cyIsImNvbnRlbnQiOiJpbXBvcnQgXCJAbWF0ZXJpYWwvd2ViL2FsbFwiIn0seyJuYW1lIjoiaW5kZXguaHRtbCIsImNvbnRlbnQiOiI8IURPQ1RZUEUgaHRtbD5cbjxoZWFkPlxuICA8c2NyaXB0IHR5cGU9XCJtb2R1bGVcIiBzcmM9XCIuL3NpbXBsZS1ncmVldGluZy5qc1wiPjwvc2NyaXB0PlxuPC9oZWFkPlxuPGJvZHk-XG4gIDxtZC1zbGlkZXI-PC9tZC1zbGlkZXI-XG4gIFxuICA8c2NyaXB0PlxuICAgIGNvbnN0IHNsaWRlciA9IGRvY3VtZW50LnF1ZXJ5U2VsZWN0b3IoXCJtZC1zbGlkZXJcIik7XG4gICAgc2xpZGVyLmFkZEV2ZW50TGlzdGVuZXIoXCJjaGFuZ2VcIiwgKGUpID0-IGNvbnNvbGUubG9nKGUpKTtcbiAgICBzbGlkZXIuYWRkRXZlbnRMaXN0ZW5lcihcImlucHV0XCIsICgpID0-IGNvbnNvbGUubG9nKFwiSW5wdXRcIikpO1xuICA8L3NjcmlwdD5cbjwvYm9keT5cbiJ9LHsibmFtZSI6InBhY2thZ2UuanNvbiIsImNvbnRlbnQiOiJ7XG4gIFwiZGVwZW5kZW5jaWVzXCI6IHtcbiAgICBcImxpdFwiOiBcIl4zLjAuMFwiLFxuICAgIFwiQGxpdC9yZWFjdGl2ZS1lbGVtZW50XCI6IFwiXjIuMC4wXCIsXG4gICAgXCJsaXQtZWxlbWVudFwiOiBcIl40LjAuMFwiLFxuICAgIFwibGl0LWh0bWxcIjogXCJeMy4wLjBcIlxuICB9XG59IiwiaGlkZGVuIjp0cnVlfV0). I agree the docs sure could have used some improvement though.

**Comment by simonh1000** [comment]
hm, yes, you are right. the events do fire. Will look to the elm community for why I can't detect them
